### ISSUE TITLE
"Pagination" of T under a parent

### ISSUE THREAD

**Issue by adammertel** [DISSINET/InkVisitor#1436]
Find a UI method to handle a large number of subT in the T tree

**Comment by davidzbiral** [comment]
There can be up to 1000 items in a "folder".

**Comment by davidzbiral** [comment]
Probably also make root T part of the same solution - this would solve #389.

**Comment by Ptrhnk** [comment]
Good idea, this would help us with optimisation as well.

**Comment by davidzbiral** [comment]
As a short-term solution, I am thinking about using the current filtering of T tree, but as default view, (1) unfolding the filter dropdown and (2) by default, showing not not all, but only those which are either non-empty or starred (or something like that). 
Related to #2514.

**Comment by Ptrhnk** [comment]
@davidzbiral Would you still like to have stared and non-empty Ts as a default in the tree?

> As a short-term solution, I am thinking about using the current filtering of T tree, but as default view, (1) unfolding the filter dropdown and (2) by default, showing not not all, but only those which are either non-empty or starred (or something like that). Related to [#2514](https://github.com/DISSINET/InkVisitor/issues/2514).

**Comment by davidzbiral** [comment]
@Ptrhnk Yes, unless you implement pagination very soon, I'd like to make this the default view, but with unfolding all perhaps more clearly visible than now.

**Comment by Ptrhnk** [comment]
@davidzbiral I managed to implement it, I'm going to release it with other features this week.

**Comment by Ptrhnk** [comment]
@davidzbiral FYI The pagination currently looks like this. I've also added count of children into the tooltip.

<img width="557" height="436" alt="Image" src="https://github.com/user-attachments/assets/348b2783-7a7f-4909-b847-ec8b1e85513b" />
